### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by aksanakandr**
Task_2

### CODE REVIEW

File: task2.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# The English language has five vowels: A, E, I, O, and U
```
Comment (antohaUa) [inline task2.py:1]: Still present linter errors:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py
./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py:1:1: D100 Missing docstring in public module
./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py:2:80: E501 line too long (84 > 79 characters)
./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py:22:38: W291 trailing whitespace
./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py:58:13: C812 missing trailing comma
./T-HOMETASKS/L02/l02_h01_AksanaKandratovich.py:63:17: W292 no newline at end of file

File: task2.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# | vowel | count |
+# -----------------
+# |   a   |  11   |
+# |   e   |  23   |
+#   ...
+# -----------------
+
+# then modify text where each vowel replaced with
+# A->À;  a->à ; E-> É ; e->é; I->Í , i->í ; O->Ó ; o->ó; U->Ú; u->ú
+# ex. "Í wàndéréd lónély...."   and print it
```
Comment (antohaUa) [inline task2.py:14]: All comments at the top of the module better organize as docstring. Smth aka
"""
Hometask02.

Task description ...
"""

File: task2.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Tossing their heads in sprightly dance."""
+
+vowel_counts = {'a': 0, 'e': 0, 'i': 0, 'o': 0, 'u': 0}
+poem_text_lower = poem_text.lower()
+
+for char in poem_text_lower:
+    if char in vowel_counts:
+        vowel_counts[char] += 1
+
+sep_num = 20
```
Comment (antohaUa) [inline task2.py:38]: This better to store as constant ( variable with uppercase naming) aka SEP_NUM = 20
Constants always better to place at the top of the file right after docstring and imports

File: task2.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+vowel_counts = {'a': 0, 'e': 0, 'i': 0, 'o': 0, 'u': 0}
+poem_text_lower = poem_text.lower()
+
+for char in poem_text_lower:
```
Comment (antohaUa) [inline task2.py:34]: A little bit complicated. Better to iterate via vowels and do poem_text_lower.count(current_vowel)

File: task2.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+print('-' * sep_num)
+print(f"| {'vowel':^5} | {'count':^8} |")
+print('-' * sep_num)
+
+for vowel, count in vowel_counts.items():
+    print(f'| {vowel:<5} | {count:^8} |')
+
+print('-' * sep_num)
+
+replacements = {
```
Comment (antohaUa) [inline task2.py:48]: This also better to store as constant

**Review by antohaUa — APPROVED** [review]

File: task2_new_format.py
Comment (aksanakandr) [inline task2_new_format.py:1]: @antohaUa 
please check my update

File: task2_new_format.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
+"""The English language has five vowels: A, E, I, O, and U.
+
+# Please count each vowel occurence in text bellow ( sum of lower and capital cases)
+# and write output as table smth like this
```
Comment (antohaUa) [inline task2_new_format.py:4]:
```suggestion
"""
Hometask02 String operations.

The English language has five vowels: A, E, I, O, and U.
Please count each vowel occurence in text bellow ( sum of lower and capital cases)
and write output as table smth like this
```

**Review by antohaUa — APPROVED** [review]

**Pull request merged**